### PR TITLE
Add USDC to Lido - Review

### PR DESCRIPTION
**Pull request by marczeller**

### CODE REVIEW

File: src/20241002_AaveV3EthereumLido_OnboardUSDCToAaveV3LidoInstance/AaveV3EthereumLido_OnboardUSDCToAaveV3LidoInstance_20241002.sol [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    );
+  }
+
+  function newListings() public pure override returns (IAaveV3ConfigEngine.Listing[] memory) {
+    IAaveV3ConfigEngine.Listing[] memory listings = new IAaveV3ConfigEngine.Listing[](1);
+
+    listings[0] = IAaveV3ConfigEngine.Listing({
+      asset: USDC,
+      assetSymbol: 'USDC',
+      priceFeed: 0x736bF902680e68989886e9807CD7Db4B3E015d3C,
```
Comment (sakulstra) [inline src/20241002_AaveV3EthereumLido_OnboardUSDCToAaveV3LidoInstance/AaveV3EthereumLido_OnboardUSDCToAaveV3LidoInstance_20241002.sol:38]: better use AaveV3EthereumAssets.USDC_ORACLE

File: src/20241002_AaveV3EthereumLido_OnboardUSDCToAaveV3LidoInstance/AaveV3EthereumLido_OnboardUSDCToAaveV3LidoInstance_20241002.sol [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      eModeCategory: AaveV3EthereumLidoEModes.NONE,
+      enabledToBorrow: EngineFlags.ENABLED,
+      stableRateModeEnabled: EngineFlags.DISABLED,
+      borrowableInIsolation: EngineFlags.DISABLED,
```
Comment (sakulstra) [inline src/20241002_AaveV3EthereumLido_OnboardUSDCToAaveV3LidoInstance/AaveV3EthereumLido_OnboardUSDCToAaveV3LidoInstance_20241002.sol:42]: while there is nothing in isolation (yet) usually stables are - does it make sense to enable?

File: src/20241002_AaveV3EthereumLido_OnboardUSDCToAaveV3LidoInstance/OnboardUSDCToAaveV3LidoInstance.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+## Specification
+
+The table below illustrates the configured risk parameters for **USDC**
+
+| Parameter                 |                                      Value |
+| ------------------------- | -----------------------------------------: |
+| Isolation Mode            |                                      false |
+| Borrowable                |                                    ENABLED |
+| Collateral Enabled        |                                       true |
```
Comment (sakulstra) [inline src/20241002_AaveV3EthereumLido_OnboardUSDCToAaveV3LidoInstance/OnboardUSDCToAaveV3LidoInstance.md:28]: false
```suggestion
| Collateral Enabled        |                                       false |
```

File: src/20241002_AaveV3EthereumLido_OnboardUSDCToAaveV3LidoInstance/OnboardUSDCToAaveV3LidoInstance.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+| Liquidation Protocol Fee  |                                       10 % |
+| Reserve Factor            |                                       10 % |
+| Base Variable Borrow Rate |                                        0 % |
+| Variable Slope 1          |                                      5.5 % |
+| Variable Slope 2          |                                       60 % |
+| Uoptimal                  |                                       92 % |
+| Stable Borrowing          |                                   DISABLED |
+| Flashloanable             |                                    ENABLED |
+| Siloed Borrowing          |                                   DISABLED |
+| Borrowable in Isolation   |                                   DISABLED |
```
Comment (sakulstra) [inline src/20241002_AaveV3EthereumLido_OnboardUSDCToAaveV3LidoInstance/OnboardUSDCToAaveV3LidoInstance.md:44]: should it though?

File: src/20241002_AaveV3EthereumLido_OnboardUSDCToAaveV3LidoInstance/OnboardUSDCToAaveV3LidoInstance.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+| Oracle                    | 0x736bF902680e68989886e9807CD7Db4B3E015d3C |
```
Comment (sakulstra) [inline src/20241002_AaveV3EthereumLido_OnboardUSDCToAaveV3LidoInstance/OnboardUSDCToAaveV3LidoInstance.md:45]:
```suggestion
| Oracle                    | [0x736bF902680e68989886e9807CD7Db4B3E015d3C](https://etherscan.io/address/0x736bF902680e68989886e9807CD7Db4B3E015d3C) |
```

**Review by sendra — APPROVED** [review]

**Pull request merged**